app/api/api_v1/endpoints/assignment_router.py

```diff
@@ -24,6 +25,7 @@ class UpdateAssignmentBody(BaseModel):
     max_attempts: PositiveInt | None
     available_date: datetime | None
     due_date: datetime | None
+    is_published: bool = UNSET
```

Comment:
this isn't used

Reply:
<img width="484" alt="image" src="https://github.com/user-attachments/assets/2d542b6b-ca12-4e81-8361-66c93fe07b82">